### PR TITLE
Add mongoProc.app v3.0

### PR DESCRIPTION
**Pull request by kolomiichenko**
More details about application: https://university.mongodb.com/mongoproc
And I not found information about license

### CODE REVIEW

File: Casks/mongoproc.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'mongoproc' do
+  version '3.0'
+  sha256 :no_check
+
+  url 'https://s3.amazonaws.com/edu-downloads.10gen.com/3.0/mongoProc_3.0_mac.dmg'
```
Comment (fanquake) [inline Casks/mongoproc.rb]: You can use #{version} in place of both version numbers in the URL.

File: Casks/mongoproc.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'mongoproc' do
+  version '3.0'
+  sha256 :no_check
```
Comment (fanquake) [inline Casks/mongoproc.rb]: You need to use a sha256 here.

**Comment by kolomiichenko** [comment]
@fanquake, complete :smiley:

**Comment by kolomiichenko** [comment]
@Ngrd, Complete. 
When l can read all this recommendations (for next time)?

**Comment by Ngrd** [comment]
Thank you for all the changes and your patience.

The recommendations can be found first of all in [CONTRIBUTING.md](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md). Additionally, plenty of other useful things can be found in the [CASK_LANGUAGE_REFERENCE.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md).

**Comment by Ngrd** [comment]
Thank you for the contribution. It was merged directly as 2e337e2ec2cc3d44485349a564e39e316e61e7e5 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**